.github/ISSUE_TEMPLATE/1-documentation.md

```diff
@@ -9,16 +9,16 @@ assignees: ''
 
 <!--
 
-Welcome!  Unfortunately not all documentation is perfect, and if you're opening
+Welcome!Unfortunately, not all documentation is perfect, and if you're opening
```

Comment:
There's not a need for the comma after `Unfortunately`, and I'm not sure why you've removed the space between `Welcome!` and `Unfortunately`.